### PR TITLE
doc improvment

### PR DESCRIPTION
**Pull request by Biki-das**
small docs improvment

### CODE REVIEW

File: content/en/docs/Overview/_index.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ It can also work as a lightweight Subsonic-API compatible server, that can be us
 - Streams virtually **any audio format** available
 - Reads and uses all your beautifully curated **metadata**
 - Great support for **compilations** (Various Artists albums) and **box sets** (multi-disc albums)
-- **Multi-user**, each user has their own play counts, playlists, favorites, etc..
-- **Multi-platform**, runs on macOS, Linux and Windows. **Docker** images are also provided
+- **Multi-user**, each user has their own play counts, playlists, favorites, etc.
```
Comment (deluan) [inline content/en/docs/Overview/_index.md:22]: This actually should be:
> playlists, favorites, etc...

meaning that the list is not complete

File: content/en/docs/Overview/_index.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ trouble with the client of your choice.
 ## Road map
 
 This project is in active development. Expect a more polished experience and new features/releases 
-on a frequent basis. Some upcoming features planned: 
+on a frequently. Some upcoming features planned:
```
Comment (deluan) [inline content/en/docs/Overview/_index.md:81]: I think the two are synonymous, no?

**Review by deluan — COMMENTED** [review]
Thanks, English is not my first language, so any help proofreading the docs is welcome. Can you check my comments below?

File: content/en/docs/Overview/_index.md (hunk 2 above)
Comment (Biki-das) [reply content/en/docs/Overview/_index.md:81]: Yeah but frequent is more appropriate here

**Comment by Biki-das** [comment]
@deluan navidrome seems interesting to me, but i am a beginner, i have past contributions to open source, but i am just confused on where should i start like i need to have a thorough understanding of the codebase to solve bugs and add new features, so any guidance from you will be highly appreciated

File: content/en/docs/Overview/_index.md [hunk 3]
```diff
@@ -66,22 +66,20 @@ Besides its own Web UI, Navidrome should be compatible with all Subsonic clients
...
+
+For more options, look at the [list of clients](https://airsonic.github.io/docs/apps/) maintained by
+the Airsonic project. Please open an [issue](https://github.com/navidrome/navidrome/issues) if you have any
 trouble with the client of your choice.
 
 ## Road map
 
-This project is in active development. Expect a more polished experience and new features/releases 
-on a frequent basis. Some upcoming features planned: 
+This project is in active development. Expect a more polished experience and new features/releases frequently. Some upcoming features planned:
```
Comment (deluan) [inline content/en/docs/Overview/_index.md:80]: I still prefer _on a frequent basis_, unless it is wrong English....

**Comment by deluan** [comment]
> @deluan navidrome seems interesting to me, but i am a beginner, i have past contributions to open source, but i am just confused on where should i start like i need to have a thorough understanding of the codebase to solve bugs and add new features, so any guidance from you will be highly appreciated

You should start by setting up your dev environment, being able to compile and run locally using `make dev`. Then browse the GitHub issues, filtered by [good first issue](https://github.com/navidrome/navidrome/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and try to understand the code. Any further questions, feel free to ask on our #dev channel in Discord, or on the issue you are investigating.

File: content/en/docs/Overview/_index.md (hunk 3 above)
Comment (Biki-das) [reply content/en/docs/Overview/_index.md:80]: Ok

**Comment by deluan** [comment]
No news for some time, closing.

**Pull request closed without merging**